qa/tasks/cephfs/test_admin.py

```diff
@@ -179,8 +179,8 @@ def gen_health_warn_mds_trim(self):
         self.config_set('mds', 'mds_debug_subtrees', 'true')
         # this will really really slow the trimming, so that MDS_TRIM stays
         # for longer.
-        self.config_set('mds', 'mds_log_trim_decay_rate', '60')
-        self.config_set('mds', 'mds_log_trim_threshold', '1')
+        self.config_set('mds', 'mds_log_max_segments', '0')
+        self.config_set('mds', 'mds_log_warn_factor', '1')
```

Comment:
Is this really required since max_sements*warn_factor would be 0 anyway?

Reply:
I don't get your point. Do you mean is setting `mds_log_max_segments` to 0 and `mds_log_warn_factor` to 1 really required? It was suggested by you for generating MDS_TRIM health warning.

Reply:
Have you checked the code in Beacon.cc where these configs are useD?

Reply:
> I don't get your point. Do you mean is setting `mds_log_max_segments` to 0 and `mds_log_warn_factor` to 1 really required? It is required to generate MDS_TRIM health warning and was method was suggested by you.

I suggested that there are two configs that control MDS_TRIM warning: max_segments and warn_factor. If there aren't any existing qa code that use this to generate trim warning, the next thing to do is to check the code to see under what circumstances the warning is generated.

```
  // Detect MDS_HEALTH_TRIM condition
  // Indicates MDS is not trimming promptly
  {
    const auto log_max_segments = mds->mdlog->get_max_segments();
    const auto log_warn_factor = g_conf().get_val<double>("mds_log_warn_factor");
    if (mds->mdlog->get_num_segments() > (size_t)(log_max_segments * log_warn_factor)) {
      CachedStackStringStream css;
      *css << "Behind on trimming (" << mds->mdlog->get_num_segments()
	<< "/" << log_max_segments << ")";

      MDSHealthMetric m(MDS_HEALTH_TRIM, HEALTH_WARN, css->strv());
      m.metadata["num_segments"] = stringify(mds->mdlog->get_num_segments());
      m.metadata["max_segments"] = stringify(log_max_segments);
      health.metrics.push_back(m);
    }
  }
```

Now do you see the usage of the configs?

Reply:
Got, thanks.

Reply:
@rishabh-d-dave Any update on this? Do you plan to update this change with the above suggestion?

Reply:
The values are already correct as per the above suggestion.

Reply:
Didn't we discuss that setting `mds_log_warn_factor` isn't required since `mds_log_max_segments` is being set to zero?

Reply:
Sorry, this is not making sense @rishabh-d-dave.

Anyway, not a show stopper - I'll have this run through QA.